tests/cupyx_tests/jit_tests/test_raw.py

```diff
@@ -103,6 +104,7 @@ def f(x, y):
         expected = x.reshape(32, 32)[:, ::-1].ravel()
         assert bool((y == expected).all())
 
+    #@unittest.skipIf(cupy.cuda.runtime.is_hip, 'segfault?')
     def test_shared_memory_dynamic(self):
```

Comment:
This test passed on Radeon VII pro + ROCm 4.0.0, but segfaults on Radeon VII + ROCm 3.5.0. I suspect it's due to either resource allocation, which could be constrained by `__launch_bounds__`, or bugs that are fixed in recent ROCm, so let's just keep a note and not touch them in this PR.